Source/WebKit/Shared/RemoteLayerTree/RemoteScrollingUIState.h

```diff
@@ -50,8 +50,8 @@ class RemoteScrollingUIState {
 public:
     using Changes = RemoteScrollingUIStateChanges;
 
-    void encode(IPC::Encoder&) const;
-    static std::optional<RemoteScrollingUIState> decode(IPC::Decoder&);
+    RemoteScrollingUIState() = default;
```

Comment:
Do we still need this?

Reply:
Yes, since I'm adding a constructor that takes parameters, I need to explicitly define the default constructor to avoid breaking the build.